ci/request-reviews/request-code-owner-reviews.sh

```diff
@@ -77,20 +77,6 @@ if ! "$SCRIPT_DIR"/verify-base-branch.sh "$tmp/nixpkgs.git" "$headRef" "$baseRep
     exit 1
 fi
 
-log "Getting code owners to request reviews from"
-"$SCRIPT_DIR"/get-reviewers.sh "$tmp/nixpkgs.git" "$ownersFile" "$baseBranch" "$headRef" | \
-    "$SCRIPT_DIR"/process-reviewers.sh "$baseRepo" "$prNumber" "$prAuthor" > "$tmp/reviewers.json"
-
-log "Requesting reviews from: $(<"$tmp/reviewers.json")"
-
-if ! response=$(effect gh api \
```

Comment:
The one thing this doesn't preserve yet is the `effect` part here, which [implements dry mode](https://github.com/NixOS/nixpkgs/blob/17d9254ad94961ff30c72a22a2b2bba0a045d832/ci/request-reviews/request-reviews.sh#L14-L20), which is used to avoid requesting reviews from code owners for draft PRs.

Reply:
The fixups look good to me!

> Should we enable DRY_MODE for maintainer requests, too?

Yeah definitely. Will involve mirroring https://github.com/NixOS/nixpkgs/blob/a1945f760a8fe019a4d753808de424dcd4e5b3cf/.github/workflows/codeowners-v2.yml#L27 and https://github.com/NixOS/nixpkgs/blob/a1945f760a8fe019a4d753808de424dcd4e5b3cf/.github/workflows/codeowners-v2.yml#L34-L35. Could be done in a separate PR though